requirements-dev.txt

```diff
@@ -2,3 +2,5 @@
 -e ./docs/mkdocs_include_markdown_plugin
 pytest
 mkdocs==1.0.4
+pip-tools
+requests
```

Comment:
Maybe also pin all packages here?

Reply:
This PR is about build-time determinism. that said, it probably can't hurt. Do you guys use per-project virtualenvs?

Reply:
As long as things "just work", no :-/ But I think that when it comes to the docs, I do have the custom extension installed in a separate virtualenv.

Reply:
I'll leave as-is, then